ci-scripts/unit-tests.sh

```diff
@@ -76,6 +76,14 @@ echo "Running schematics unit tests and code coverage for tracking library"
 
 npm --prefix feature-libs/tracking run test:schematics -- --coverage
 
+echo "Running unit tests and code coverage for pickup-in-store library"
+
+ng test pickup-in-store --source-map --watch=false --code-coverage --browsers=ChromeHeadless
```

Comment:
can you check other libs. we use `--no-watch` instead of `--watch=false`.  check this: https://angular.io/cli#cli-command-language-syntax

Reply:
done